docs-2.0/3.ngql-guide/3.data-types/1.numeric.md

```diff
@@ -0,0 +1,13 @@
+# Numeric types
+
+## Integer
+
+An integer is declared with keyword `int` , which is 64-bit *signed*. The supported range is [-9223372036854775808, 9223372036854775807]. Integer constants support multiple formats:
+
+1. Decimal, for example `123456`
+2. Hexadecimal, for example `0xdeadbeaf`
+3. Octal, for example `01234567`
+
+## Double floating point
+
+Double floating point data type is used for storing double precision floating point values. The keyword used for double floating point data type is `double`. There are no upper and lower ranges.
```

Comment:
Spare the first sentence because anyone who knows what a double-precision floating-point number is would think it redundant, and for those who don't know what that is, it is just as its name implies.

Reply:
ditto